index.html

```diff
@@ -151,29 +151,31 @@ <h2>
     </section>
     <section>
       <h2>
-        Interface definitions
-      </h2>
-      <section>
-        <h2>
-          Extensions to the <a>Screen</a> interface
-        </h2>The CSSOM View specification defines a <code>Screen</code>
-        interface [[!CSSOM-VIEW]], which this specification extends:
-        <pre class='idl'>
+        Extensions to the <a>Screen</a> interface
+      </h2>The <a data-cite="CSSOM-View">CSSOM View Module</a> specification
+      defines a <code>Screen</code> interface, which this specification
+      extends:
+      <pre class='idl'>
           partial interface Screen {
             [SameObject] readonly attribute ScreenOrientation orientation;
           };
         </pre>
+      <section>
+        <h2>
+          <dfn data-dfn-for='Screen'>orientation</dfn> attribute
```

Comment:
I'd suggest moving `data-dfn-for='Screen'` up to the `<section>` tag on line 152. That way, it's clear that the whole section is defining 'Screen''s things (even though there is only one attribute).